Gemfile.lock

```diff
@@ -2,7 +2,6 @@ PATH
   remote: .
```

Comment:
If this file was added to the `.gitignore`, should we just remove it?

Reply:
Yes.

I was originally going to do that, but [Bundler's current best practices](https://bundler.io/guides/faq.html#using-gemfiles-inside-gems) includes committing `Gemfile.lock` for a better first-run experience for new contributors.

Reply:
Ok! But then should we take it back out `.gitignore`?

Reply:
I did remove it from `.gitignore`, didn't I?

Reply:
🤦‍♂ I read the diff backwards, I'm so sorry. I'll merge this